libs/sdk-ui-dashboard/src/dashboard/Dashboard.tsx

```diff
@@ -295,15 +315,40 @@ export interface IDashboardProps {
     children?: JSX.Element | ((dashboard: any) => JSX.Element);
 }
 
+const menuItems: MenuButtonItem[] = [
+    {
+        itemId: "save-as-new",
+        itemName: "Save as new",
+        callback: () => {
+            console.log("Save as new clicked");
+        },
+    },
+    {
+        itemId: "sep-1",
+        itemName: "Separator",
+        callback: noop,
```

Comment:
Maybe the `callback` prop should be optional? It could help styling: items with callback would have cursor: pointer, items without callback would have normal cursor.